modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/PersistenceCommand.java

```diff
@@ -58,7 +59,7 @@ public class PersistenceCommand extends AbstractCommand<PersistenceArguments> {
     /** {@inheritDoc} */
     @Override public Object execute(GridClientConfiguration clientCfg, Logger logger) throws Exception {
         try (GridClient client = Command.startClient(clientCfg)) {
-            Optional<GridClientNode> firstNodeOpt = client.compute().nodes().stream().findFirst();
+            Optional<GridClientNode> firstNodeOpt = anyConnectableServerNode(client.compute());
```

Comment:
'before' a non-connectable node might be chosen, so the change is not equivalent. Is this a fix?

Reply:
In the case of persistence command the whole cluster contains only one node as persistence command works only with nodes in Maintenance Mode for persistence repair and clean-up.

So here it doesn't matter.